src/js/helpers/Util.js

```diff
@@ -1,6 +1,6 @@
 var consecutiveNumber = 0;
 var objectPath = require("object-path");
-
+var toString = Object.prototype.toString;
```

Comment:
What does this do? I think this needs a comment.

Could you add a newline above?